locale/sw.yml

```diff
@@ -1396,42 +1396,42 @@ helper: "" # (Not currently supported by ASP.NET Core)
 # |   LANGUAGE: RUBY   |
 # ----------------------
 # << Keywords >>
-alias: ""
-and: ""
-begin: ""
-break: ""
-case: ""
-class: ""
-def: ""
-defined?: ""
-do: ""
-else: ""
-elsif: ""
-end: ""
-ensure: ""
-false: ""
-for: ""
-if: ""
-in: ""
-module: ""
-next: ""
-nil: ""
-not: ""
-or: ""
-redo: ""
-rescue: ""
-retry: ""
-return: ""
-self: ""
-super: ""
-then: ""
-true: ""
-undef: ""
-unless: ""
-until: ""
-when: ""
-while: ""
-yield: ""
+alias: "jina-mbadala"
+and: "na"
+begin: "anza"
+break: "vunja"
+case: "kisa"
+class: "darasa"
+def: "fanya"
+defined?: "imefafanuliwa?"
+do: "fanya"
+else: "au"
+elsif: "au-kama"
+end: "maliza"
+ensure: "hakikisha"
+false: "uwongo"
```

Comment:
Thank you for contributing the keyword translations for Ruby and Rust.  Could you update the format to the new style for your translations? This will make it easier for people to add their review for the translation.
```
  False: "uwongo"
    # ✅ @collinskoech11 
```